From fd2f2c779acd04ae42aa6c91b6f2eb3403112637 Mon Sep 17 00:00:00 2001
From: yxj <yxj@rock-chips.com>
Date: Thu, 27 Dec 2012 17:02:26 +0800
Subject: [PATCH] rkfb_sys.c:fix warning 	warning: initialization
 discards qualifiers from pointer target type 	warning: no return statement
 in function returning non-void

---
 drivers/video/rockchip/rkfb_sysfs.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/drivers/video/rockchip/rkfb_sysfs.c b/drivers/video/rockchip/rkfb_sysfs.c
index 67ba6c9f8db7..5714cf10d6a2 100644
--- a/drivers/video/rockchip/rkfb_sysfs.c
+++ b/drivers/video/rockchip/rkfb_sysfs.c
@@ -275,13 +275,14 @@ static ssize_t set_fb_win_map(struct device *dev,struct device_attribute *attr,
 static ssize_t show_dsp_lut(struct device *dev,
 		struct device_attribute *attr, char *buf)
 {
-	
+
+	return 0;	
 }
 static ssize_t set_dsp_lut(struct device *dev,struct device_attribute *attr,
 	const char *buf, size_t count)
 {
 	int dsp_lut[256];
-	char *start = buf;
+	const char *start = buf;
 	int i=256,j,temp;
 	int space_max = 10;
 
-- 
2.35.3

